system/COPY/etc/httpd/conf.d/cfme-https-application.conf

```diff
@@ -1,6 +1,7 @@
 ## CFME SSL Virtual Host Context
 
 <VirtualHost *:443>
+KeepAlive On
```

Comment:
@martinpovolny Please change this one to lower case, I see many other options using all lower case, such as `SSLEngine on`, `LogLevel warn`.  Even if it doesn't matter, I'd rather we were consistent.